Tests/js/views/services/assets/ez-roleserversideviewservice-tests.js

```diff
@@ -632,8 +633,61 @@ YUI.add('ez-roleserversideviewservice-tests', function (Y) {
         }
     });
 
+    refreshAssignmentsTabTest = new Y.Test.Case({
+        name: "eZ Role Server Side View Service refresh assignments tab test",
+
+        setUp: function () {
+            this.capi = new Mock();
+            this.app = new Mock();
+            this.roleId = 42;
+
+            this.service = new Y.eZ.RoleServerSideViewService({
+                capi: this.capi,
+                app: this.app,
+            });
+
+            Mock.expect(this.service, {
```

Comment:
`this.service` is not mock object so it shouldn't be proceeded like it.
Besides that `Y.eZ.RoleServerSideViewService` extends `Y.Base` so if it is initiated like above, then `this.service.get('app')` should return the value of attribute passed to the constructor.